packages/react-router/test/base/src/pages/nested-tabs-relative-links/NestedTabsRelativeLinks.tsx

```diff
@@ -0,0 +1,194 @@
+import {
+  IonContent,
+  IonHeader,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  IonRouterOutlet,
+  IonTabs,
+  IonTabBar,
+  IonTabButton,
+  IonIcon,
+  IonLabel,
+  IonBackButton,
+  IonButtons,
+} from '@ionic/react';
+import { triangle, ellipse, square } from 'ionicons/icons';
+import React from 'react';
+import { Link, Navigate, Route } from 'react-router-dom';
+
+/**
+ * This test page verifies that relative links work correctly within
+ * nested IonRouterOutlet components, specifically in a tabs-based layout.
+ *
+ * Issue: When using React Router's <Link to="relative"> inside nested
+ * outlets with index routes, the relative path resolution can produce
+ * incorrect URLs (e.g., /tab1/tab1/abc instead of /tab1/abc).
```

Comment:
Nit: it might be clearer here if you use the same path name in the `to` field on `Link` as you do in the example (`relative` vs `abc`) and say that you are inside of the `tab1` route when navigating.